uniffi_core/release.toml

```diff
@@ -5,12 +5,12 @@
 #
 # All other config exists in [../release.toml](../release.toml).
 
-tag = true
+tag = false
 
 # This is how we manage the sections in CHANGELOG.md
 pre-release-replacements = [
-  {file="../CHANGELOG.md", search="\\[\\[UnreleasedBackendVersion\\]\\]", replace="v{{version}}", exactly=2},
+  {file="../CHANGELOG.md", search="\\[\\[UnreleasedBackendVersion\\]\\]", replace="v{{version}}", exactly=1},
   {file="../CHANGELOG.md", search="\\[\\[ReleaseDate\\]\\]", replace="{{date}}", exactly=1},
   {file="../CHANGELOG.md", search="\\.\\.\\.HEAD\\)", replace="...{{tag_name}})", exactly=1},
```

Comment:
I see the `{{tag_name}}` part here. That worked before I guess, why didn't it work now? hm.